app/process/watcher/evm/watcher.go

```diff
@@ -178,20 +181,23 @@ func NewWatcher(
 		targetBlock = uint64(startBlock)
 		log.Tracef("[%s] - Updated Transfer Watcher timestamp to [%s]", dbIdentifier, timestamp.ToHumanReadable(startBlock))
 	}
-	return &Watcher{
-		repository:        repository,
-		dbIdentifier:      dbIdentifier,
-		contracts:         contracts,
-		prometheusService: prometheusService,
-		pricingService:    pricingService,
-		evmClient:         evmClient,
-		logger:            c.GetLoggerFor(fmt.Sprintf("EVM Router Watcher [%s]", dbIdentifier)),
-		assetsService:     assetsService,
-		targetBlock:       targetBlock,
-		validator:         validator,
-		sleepDuration:     pollingInterval,
-		filterConfig:      filterConfig,
-	}
+	istance := &Watcher{
```

Comment:
Why do we need the variable?

Reply:
It is easier to debug this way. Should I remove it?

Reply:
removed